modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/layouts/03_company/page-definition.json

```diff
@@ -283,10 +283,10 @@
 												"borderRadius": "",
```

Comment:
Please rename this folder to `03_organization` and any other "company" to "organization"